IntelliTect.TestTools.WindowsTestWrapper/CustomDictionary.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version='1.0' encoding='utf-8' ?>
+
+<Dictionary>
+
+  <Words>
+
+    <Recognized>
+
+      <Word>Intelli</Word>
+      <Word>Tect</Word>
+
+    </Recognized>
+
+  </Words>
+
+</Dictionary>
```

Comment:
It appears that there are multiple copies of this same dictionary. Rather than having one specific to each project, what about putting this file along side the solution file, and simply referencing it from both projects (ie. `<CodeAnalysisDictionary Include="..\CustomDictionary.xml" />`